EIPS/eip-4750.md

```diff
@@ -66,8 +66,9 @@ We introduce two new instructions:
 
 1. `CALLF` (`0xb0`)
 2. `RETF` (`0xb1`)
+3. `TAILCALLF` (`0xb2`)
```

Comment:
I think this should actually be called `JUMPF` to signal that we merely move to another function, but keep the return stack intact.

However, we do need to perform the stack cleanup in the `RETF` function, because code utilising a `CALLF -> JUMPF ... -> RETF` chain has no way of knowing how much to clean up, as it depends on the call path.